docs/data-api.md

```diff
@@ -43,6 +43,46 @@ console.log(doc._id, doc._rev, doc.title, doc.author, doc.isbn)
 ```
 
 
+### `cozy.client.data.findMany(doctype, [ids])`
+
+`cozy.client.data.findMany(doctype, [ids])` returns the documents associated to the given IDs.
+
+It returns a promise for a map, with the given IDs as keys.
+
+When a document is found, the corresponding value will be an object with a
+single `doc` key containing the document.
+Documents will have the same fields than the returned values of `create`,
+including `_id` and `_rev`.
+
+When a document is missing, the corresponding value will be an object with a
+single `error` key containing the error message.
+Even when some document is missing, the promise will still be fulfilled or the
+callback will be passed the resulting map, so you can still access the other
+documents found and identify the missing ones.
+
+- `doctype` is a string specifying the
+  [doctype](intro.md#doctypes--permissions)
+- `ids` is an array of strings specifying the identifiers of the documents you
+  look for
+
+**Warning**: Not available on **v2**. A fallback implementation could be
+provided at some point, but it would make one request per id anyway, since the
+old data system doesn't provide such an API.
```

Comment:
@nono I looked at the [old data-system cookbook](https://docs.cozy.io/en/hack/cookbooks/data-system.html) (which doesn't seem to be available right now but still was 3 days ago). The cookbook didn't include anything about fetching multiple docs from a set of ids. But I didn't look at the data-system code, so no I'm not sure :)

I can remove the last part of the comment. Or do you think we should provide the v2 implementation right now?